### ISSUE TITLE
Add file system data store

### ISSUE THREAD

**Issue by hallahan** [digidem/simple-odk#10]
Although there is a file system media store, a file system data store would also be useful. Github & S3 are great, but a simple file system store would be quite useful if collecting and serving from something like a Raspberry Pi.

Have you considered this option?

**Comment by gmaclennan** [comment]
Yes, definitely planning on this. See also [OpenMapKitServer](https://github.com/AmericanRedCross/OpenMapKitServer/tree/master/odk) which is built on Simple-ODK

**Comment by hallahan** [comment]
Yes, that's what I made after asking this question ^_^.

We could probably put in a file system store like we did on OpenMapKit Server with low effort.

**Comment by gmaclennan** [comment]
Oh, hi Nick! now I know your Github account :)
Haven't revisited simple-odk for a little while, but last time I was at it I was moving everything to streams in the [`streams`](https://github.com/digidem/simple-odk/tree/streams) branch. I should be taking a look at where that code is in the next 2-3 weeks.
